volatility3/cli/volshell/linux.py

```diff
@@ -50,6 +58,7 @@ def construct_locals(self) -> List[Tuple[List[str], Any]]:
         result += [
             (["ct", "change_task", "cp"], self.change_task),
             (["lt", "list_tasks", "ps"], self.list_tasks),
+            (["gp", "get_task"], self.get_task),
```

Comment:
How do we feel about the naming?  Should it be gt in linux, or will the difference be confusing?  Should we call it `get_process` instead to keep with windows?  We can allow alternatives (see `list_tasks` as both `lt` and `ps` in the line above).  Perhaps we should have it as `gt` and then with `gp` as a secondary?

Reply:
Ah, you're right... shame...  5:S  Would it be worth aliasing it to `get_process` as well, or do we just live with the inconsistency?